utils/update-next.js

```diff
@@ -0,0 +1,58 @@
+const { execFileSync } = require('child_process');
+
+function exec(cmd, args, opts) {
+  // eslint-disable-next-line no-console
+  console.log(`${cmd} ${args.join(' ')}`);
+  return execFileSync(cmd, args, opts);
+}
+
+module.exports = async ({ github, context }) => {
+  const oldVersion = require('../examples/nextjs/package.json').dependencies
+    .next;
+  const newVersion = exec('npm', ['view', 'next', 'dist-tags.latest']);
+  const branch = `next-${newVersion.replaceAll('.', '-')}`;
+
+  if (oldVersion === newVersion) {
+    // eslint-disable-next-line no-console
+    console.log(
+      `Next.js version ${newVersion} did not change, skipping update.`
+    );
+    return;
+  }
+
+  if (
+    exec('git', ['ls-remote', '--heads', 'origin', branch]).toString().trim()
+  ) {
+    // eslint-disable-next-line no-console
+    console.log(`Branch ${branch} already exists, skipping update.`);
+    return;
+  }
+
+  exec('rm', ['-rf', './examples/nextjs']);
+  exec('npx', ['--yes', 'create-next-app@latest', './examples/nextjs']);
```

Comment:
Is this example only ever the exact output of `create-next-app`? We don't want any tweaks from that to be in our example?